docs/modules/ROOT/pages/connect-to-viridian.adoc

```diff
@@ -1,30 +1,49 @@
 == Connecting to {hazelcast-cloud} with Hazelcast CLC
-:description: To use the Hazelcast CLC with {hazelcast-cloud}, you can download a preconfigured client. You don’t need to do any configuration.
+:description: To use the Hazelcast CLC with {hazelcast-cloud}, you can import CLC configuration from a Viridian cluster. You don’t need to do any configuration.
 
 :page-product: cloud
 
 {description}
 
-The xref:configuration.adoc[configuration file] in a sample client provides the Hazelcast CLC with with the correct connection credentials.
+You can also create a cluster using CLC, which automatically makes the cluster configuration available. See Managing Viridian Clusters for more information.
 
 == Before you Begin
 
 You need the following:
 
 - Hazelcast CLC
 - A {hazelcast-cloud} cluster
+- Viridian API key and secret
 
-TIP: You can find all connection credentials such as the cluster name and discovery token in the {hazelcast-cloud} console. For details, see xref:cloud:ROOT:connect-to-cluster.adoc[].
+TIP: You can create and find the API key and secret in the {hazelcast-cloud} console. For details, see xref:cloud:ROOT:developer.adoc[].
 
 [[mutual]]
-== Connecting to Clusters
+== Retrieving the Viridian Token
 
-. Download the Go Client sample from {hazelcast-cloud} console.
+. In order to enable CLC to do Viridian cluster operations, a token must be generated using a Viridian API key and secret. Retrieve the Viridian token using the following command:
++
+```bash
+clc viridian login
+```
++
+Enter your API key and secret when asked. If the API key and secret is correct, the Viridian token will be retrieved and stored.
+
+[[mutual]]
+== Listing the Clusters
+
+. Run the following command in order to list the clusters your account can access:
++
+```bash
+clc viridian list-clusters
+```
+
```

Comment:
We take the status directly from the Viridian API and use verbatim. There are more statuses other than RUNNING, etc. so I think it would be better to not mention the actual statuses.